data/pokedex.js

```diff
@@ -11719,6 +11747,21 @@ exports.BattlePokedex = {
 		prevo: "yungoos",
 		evoLevel: 20,
 		eggGroups: ["Field"],
+		otherFormes: ["gumshoostotem"],
```

Comment:
What happened to [Pokemon]-Alola-Totem

Reply:
What are you talking about? Alolan Gumshoos is not a thing...

Reply:
what

Reply:
like you didnt even comment on anything relating to that you just commented to gumshoos's otherFormes line

Reply:
I swear Im an idiot